code/game/gamemodes/vampire/vampire.dm

```diff
@@ -297,19 +297,26 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 			to_chat(owner, "<span class='warning'>They've got no blood left to give.</span>")
 			break
 		if(H.stat < DEAD)
-			blood = min(20, H.blood_volume)	// if they have less than 20 blood, give them the remnant else they get 20 blood
-			bloodtotal += blood / 2	//divide by 2 to counted the double suction since removing cloneloss -Melandor0
-			bloodusable += blood / 2
+			if(!issmall(H) && !H.ckey)
```

Comment:
Doesn't the "&& !H.ckey" check here mean that vampires will ONLY get blood when they bite NPCs, and won't get anything from biting players? I've been seeing complaints about vampires not getting blood from biting crew since this PR was merged. This might be why.

Reply:
Should be H.ckey if we want the check to pass when used on players